source/beamline/SGM/SGMHexapodTransformedAxis.cpp

```diff
@@ -185,6 +200,111 @@ AMAction3 * SGMHexapodTransformedAxis::createSetParameterActions(double startPoi
 	return action;
 }
 
+AMAction3 *SGMHexapodTransformedAxis::createSetParametersActions(const QVector3D &startPoint, const QVector3D &endPoint, double deltaTime)
```

Comment:
It is only used in the SGMHexapodMapScanControllerAssembler to explicitly create all the trajectories.  With the trajectory always being "X is the trajectory, Y moves by a step, Z needs to be preserved" this will create all the actions for that specific trajectory.